components/esp_websocket_client/CHANGELOG.md

```diff
@@ -0,0 +1,108 @@
+# Changelog
+
+## [1.0.0](https://github.com/espressif/esp-protocols/commits/996fef7)
+
+### Updated
+
+- esp_websocket_client: Updated version to 1.0.0 Updated tests to run agains release-v5.0 ([996fef7](https://github.com/espressif/esp-protocols/commit/996fef7))
+- esp_websocket_client: * Error handling improved to show status code from server * Added new API `esp_websocket_client_set_headers` * Dispatches 'WEBSOCKET_EVENT_BEFORE_CONNECT' event before tcp connection ([d047ff5](https://github.com/espressif/esp-protocols/commit/d047ff5))
+- unite all tags under common structure py test: update tags under common structure ([c6db3ea](https://github.com/espressif/esp-protocols/commit/c6db3ea))
+- Adds esp_mqtt_cxx component ([1407dfc](https://github.com/espressif/esp-protocols/commit/1407dfc))
```

Comment:
not sure about this message `Adds esp_mqtt_cxx component` in scope of Websocket changes

Reply:
This commit changed internal component reference in `components/esp_websocket_client/examples/CMakeLists.txt`

But I see that it's a bit confusing. I'll delete this line manually.